python/thirdweb-ai/src/thirdweb_ai/common/utils.py

```diff
@@ -0,0 +1,12 @@
+def normalize_chain_id(
```

Comment:
was wondering if we should just keep it simple by regexp digits or just throw if `chain_id` is not is_digit?